src/shared-components/tooltip/index.tsx

```diff
@@ -96,19 +101,29 @@ export const Tooltip: React.FC<TooltipProps> = ({
     setHovered(false);
   };
 
+  const handleMouseEnter = () => {
+    if (disableHover) {
+      return;
+    }
+    setHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    if (disableHover) {
+      return;
+    }
```

Comment:
I think this is harmless for the sake of being explicit/keeping parity between enter/leave, but might not be necessary since the `hovered` will already by set to false when this function is triggered when `disableHover` is true.